example/lib/custom_build_step.ex

```diff
@@ -0,0 +1,12 @@
+defmodule ExampleCliApp.CustomBuildStep do
+  alias Burrito.Builder.Step
+  @behaviour Step
+
+  @impl Step
+  def execute(% Burrito.Builder.Context{} = context) do
+    IO.puts("Custom build step!")
+    IO.puts("Current Target: #{inspect(context.target)}")
```

Comment:
Nitpick: should we just print the target alias to avoid cluttering the output with potentially long target definitions, like if they contain a path to a local ERTS build?